frontend/src/dto/ApplyClientNumberDto.ts

```diff
@@ -53,7 +54,7 @@ export const formDataDto: FormDataDto = {
     incorporationNumber: "",
     businessName: "",
     goodStandingInd: "",
-    birthDate: "",
+    birthdate: "",
```

Comment:
Use `java.time.LocalDate` instead of `String`

Reply:
This is TypeScript :-)